README.md

```diff
@@ -30,7 +30,8 @@ Now that Docker is available, set up the environment for the composed applicatio
 POSTGRES_PASSWORD=databasepassword
 JWT_SECRET=signingsecret
 ```
-Now, in the same folder run `docker-compose up`. Docker compose will download base images and begin the container builds.  If all is well, the graphql server will be available at `http://localhost:4000/graphql` or `http://localhost:4000/graphiql`. The frontend UI will be available at `http://localhost:3000`. If you only want to run the DB and GraphQL services in docker, run `docker-compose up graphql`
+Changes done.
+Now, in the same folder run `sudo usermod -a -G docker $USER`, and then `docker-compose up`. Docker compose will download base images and begin the container builds.  If all is well, the graphql server will be available at `http://localhost:4000/graphql` or `http://localhost:4000/graphiql`. The frontend UI will be available at `http://localhost:3000`. If you only want to run the DB and GraphQL services in docker, run `docker-compose up graphql`
```

Comment:
This belongs with the Ubuntu install bullet above.  It isn't needed or valid for macOS or Windows.